### PR TITLE
[Cleanup] Comma spacing

### PR DESCRIPTION
**Pull request by Nihlus**
This PR corrects comma spacing in the codebase, enabling SA1001. Furthermore, it corrects binding generation to account for this and has rerun the binder to apply the changes globally.

### CODE REVIEW

File: src/Generator.Bind/CSharpSpecWriter.cs [hunk 1]
```diff
@@ -662,15 +662,13 @@ private string GetDeclarationString(Function f, Settings.Legacy settings)
             if (f.Parameters.HasGenericParameters)
             {
                 sb.Append("<");
-                foreach (Parameter p in f.Parameters)
```
Comment (Frassle) [inline src/Generator.Bind/CSharpSpecWriter.cs]: Worth replacing all this with `sb.Append(String.Join(", ", f.Parameters.Where(p => p.Generic))`?
Might need to add ToArray or change Bind to target .NET4.5 as well.

Comment (Nihlus) [inline src/Generator.Bind/CSharpSpecWriter.cs]: Probably not worth the bother at the moment. We can go for another look later.

**Comment by amulware** [comment]
What is the `travis_wait` change to the `.travis.yml` about?

Otherwise changes are as described and look fine to me.

**Comment by Nihlus** [comment]
@amulware Due to the longer build times imposed by StyleCop (especially for the larger rules like SA1001), the main OpenTK build draws over the Travis timeout of 10m without console output. The `travis_wait` command extends that timeout to 20m. This is a bit of a bummer, but build times are still below 5m locally, and will decrease further when we start decomposing the library.

**Comment by amulware** [comment]
Ah, I see. Not sure that should be 'hidden away' in a cosmetic PR, but that's up to you guys.

**Comment by Nihlus** [comment]
In this particular case, it is directly preventing the pull request from successfully building on Travis, as it cancels the build should the timeout be reached.

**Comment by Frassle** [comment]
You've sneaked some trailing whitespace fixes in with this as well :stuck_out_tongue: Apart from that minor point looks fine.

**Comment by Nihlus** [comment]
Busted!

**Pull request merged**